### PR TITLE
Resolves Velocidex/velociraptor#1543 Create new VQL entropy() function

### PR DESCRIPTION
**Pull request by rmakuch**
VQL entropy() function created.
Test file yet to be added.

### CODE REVIEW

File: docs/references/vql.yaml [hunk 1]
```diff
@@ -789,6 +789,15 @@
     description: Rc4 key (1-256bytes).
     required: true
   category: plugin
+- name: entropy
+  description: Calculates shannon scale entropy of a string.
+  type: Function
+  args:
+  - name: String
```
Comment (scudette) [inline docs/references/vql.yaml:796]: You actually don't need to write this by hand, just run make reference to have it automatically done.

In any case the arg name is string with a lower case.

**Review by scudette — APPROVED** [review]
Thanks this is very cool !

Maybe also add a quick test if you feel like it (in artifacts/testdata/server/testcases/functions.in.yaml) otherwise I can do it too.

File: docs/references/vql.yaml (hunk 1 above)
Comment (rmakuch) [reply docs/references/vql.yaml]: Good to know!
I will use make reference to update the file.

**Comment by rmakuch** [comment]
Make references updated multiple references, feel free to undo that. My apologies

**Pull request merged**